FreeCodeCamp-Shortcuts-Gmail.md

```diff
@@ -0,0 +1,24 @@
+# How you can get to inbox zero with gemail
```

Comment:
раз ты так крут почему  хожу  а ты  подумай у меня один должник людит
таких в очко долбить он по запаху найдет тебя ...и  ты полюбишь его )
13.05.2016 10:11 пользователь "Jonathan" <notifications@github.com> написал:

> In FreeCodeCamp-Shortcuts-Gmail.md
> <https://github.com/FreeCodeCamp/wiki/pull/927#discussion_r63141226>:
>
> > @@ -0,0 +1,24 @@
> > +# How you can get to inbox zero with gemail
>
> FreeCodeCamp-Shortcuts-Gmail.md Gmail-Shortcuts.md
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeCodeCamp/wiki/pull/927/files/d776c1ac8cffaa4922f4a0406676687fb4a727d2#r63141226>
>